docker-build/env.sh

```diff
@@ -7,3 +7,4 @@ ARCH=${ARCH:-$(arch)}
 PLATFORM=${PLATFORM:-Linux/${ARCH}}
 CONTAINER_FLAVOR=${CONTAINER_FLAVOR:-cuda}
 INVOKEAI_TAG=${REPOSITORY_NAME,,}-${CONTAINER_FLAVOR}:${INVOKEAI_TAG:-latest}
+CMD_OVERRIDE=${CMD_OVERRIDE:-'--web --host=0.0.0.0'}	       
```

Comment:
- env.sh is used to store variables shared between both scripts (build.sh and run.sh). If a var only used in one of them, they are stored directly in the script (maybe a bad habbit 🤔)
- The Default CMD is already present in the container, would not want to have a second place to maintain it